fastlane/fastlane
# [tests] stop generating rspec temp file in current directory

### Checklist


### Motivation and Context

Every time `bundle exec rspec` is run, this file was being created in the current dir:

<img width="684" alt="image" src="https://github.com/fastlane/fastlane/assets/8419048/34ac6ade-4589-4ae3-a05c-3f6c3dcee98a">

This was bothering me, so this PR fixes just that 😄 

### Description

The file will be created in the temp dir instead, and thus will be deleted when the OS deems necessary.

### Testing Steps

To test the file generation, run `bundle exec rspec pilot/spec/test_exporter_spec.rb` and/or `bundle exec rspec pilot/spec/tester_importer_spec.rb` in master.

To test this branch, modify your Gemfile as:

```ruby
gem 'fastlane', git: 'https://github.com/fastlane/fastlane.git', branch: 'rogerluan-update-circleci'
```

And run `bundle install` to apply the changes.

Then run `bundle exec rspec pilot/spec/test_exporter_spec.rb` and/or `bundle exec rspec pilot/spec/tester_importer_spec.rb` again to verify that the file isn't being created in the current dir anymore.
